apps/sync/test/unit/lazy-loader-mock.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Can you extend and use https://mxr.mozilla.org/gaia/source/shared/test/unit/mocks/mock_lazy_loader.js instead?